src/Sulu/Bundle/AdminBundle/Resources/js/views/Tabs/Tabs.js

```diff
@@ -134,8 +143,27 @@ class Tabs<T> extends React.Component<Props<T>> {
                     <TabsComponent onSelect={this.handleSelect} selectedIndex={selectedTabIndex}>
                         {this.sortedTabRoutes.map((tabRoute) => {
                             const tabTitle = tabRoute.options.tabTitle;
+                            const tabBadges = tabRoute.options.tabBadges;
+
+                            let badges = undefined;
+                            if (tabBadges) {
+                                badges = (Object.values(tabBadges): any).map((badge: BadgeType, index: number) => {
+                                    return (
```

Comment:
we check the type of the object in other places:
https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/Form.js#L280-L295
https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js#L246-L250